website/data/docs-navigation.js

```diff
@@ -6,6 +6,28 @@
 //   the landing page for the category
 
 export default [
+  {
+    category: 'install',
+    content: [
+      { category: 'quickstart' },
+      {
+        category: 'production',
+        content: [
+          'requirements',
+          'nomad-agent',
+          'reference-architecture',
+          'deployment-guide'
+        ]
+      },
+      'windows-service'
+    ]
+  },
+  { category: 'upgrade', content: ['upgrade-specific'] },
+  {
+    category: 'integrations',
+    content: ['consul-integration', 'consul-connect', 'vault-integration']
+  },
+  '-----------',
```

Comment:
I am sure you have but just to be extra cautious - have we had these reviewed by the nomad eng team as well? I will always make sure to run docs structure changes by at least one stakeholder directly on the nomad eng team.

Reply:
Thanks @jescalan - the trio is onboard with this change. 👍